client/components/home/Pickup/card.tsx

```diff
@@ -1,8 +1,17 @@
-import { Text, Button, ButtonGroup, VStack, Heading } from "@chakra-ui/react";
+import { Text, Button, ButtonGroup, VStack, Heading, HStack, Box } from "@chakra-ui/react";
```

Comment:
Upper case the first letter for standard practice of component files. same for itemTabs.tsx